manifests/kustomize/options/ui/overlays/standalone/kustomization.yaml

```diff
@@ -1,5 +1,6 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+namespace: kubeflow
```

Comment:
Why adding namespace here? that would create an opinionated installation right? are we ok with having `kubeflow` as the default upstream?